Exercise_2.py

```diff
@@ -1,21 +1,39 @@
 # Python program for implementation of Quicksort Sort 
-  
+# Time Complexity : O(n) = n log n
+# Space Complexity : O(n) = n
```

Comment:
Whenever stating time complexity in Big-O Notation, please always mention the worst case.
Time: O(n^2)
Space: O(log(n))
You can refer to this link for more information. https://www.bigocheatsheet.com